docs/src/main/asciidoc/application-configuration-guide.adoc

```diff
@@ -158,6 +158,19 @@ You can add the `greeting.suffix`, remove the other properties, change the value
 As usual, the application can be packaged using `./mvnw clean package` and executed using the `-runner.jar` file.
 You can also generate the native executable with `./mvnw clean package -Pnative`.
 
+== Programmatically access the configuration
+
+You can access the configuration programmatically,
+this can be handy when needed to do it dynamically, or in a class that is not a CDI bean so `@ConfigProperty` cannot be used.
```

Comment:
What about:
`It can be handy to achieve dynamic lookup or retrieve configured values from classes that are neither CDI beans or Jax-RS resources.`

Reply:
done